src/ServiceControl.Audit.Persistence.RavenDb/ServiceControl.Audit.Persistence.RavenDb.csproj

```diff
@@ -41,14 +41,8 @@
     <ProjectReference Include="..\ServiceControl.Audit.Persistence\ServiceControl.Audit.Persistence.csproj" />
   </ItemGroup>
 
-  <Target Name="CopyRavenStudio" BeforeTargets="AssignTargetPaths">
-    <PropertyGroup>
-      <PackageReferenceIdentity>%(PackageReference.Identity)</PackageReferenceIdentity>
-      <PackageReferenceIdentityLower>$(PackageReferenceIdentity.ToLower())</PackageReferenceIdentityLower>
-    </PropertyGroup>
-    <ItemGroup>
-      <None Include="$(NuGetPackageRoot)$(PackageReferenceIdentityLower)\%(PackageReference.Version)\tools\Raven.Studio.Html5.zip" CopyToOutputDirectory="PreserveNewest" Condition="'$(PackageReferenceIdentityLower)' == 'ravendb.database'" />
-    </ItemGroup>
-  </Target>
+  <ItemGroup>
+    <None Include="$(PkgRavenDB_Database)\tools\Raven.Studio.Html5.zip" CopyToOutputDirectory="PreserveNewest" Visible="false" />
```

Comment:
Why do we need `Visible="false"` here?

Reply:
So that the file doesn't show up in Solution Explorer needlessly.